CHANGELOG.md

```diff
@@ -60,6 +60,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Performance issue in `OpenAiChatPromptDriver` when extracting unused rate-limiting headers.
 - Streaming not working when using deprecated `Structure.stream` field.
 - Raw Tool output being lost when being executed by ActionsSubtask.
+- Enumerate Structure parent tasks and handle non-existent parent.
```

Comment:
We should specify that this fix was done in `Workflow.insert_tasks`. And were were already enumerating parent tasks, but we were using `index` which was using object equality as opposed to id equality.